### PR TITLE
fix: added random id to logos

### PR DESCRIPTION
**Pull request by Nuzhy-Deriv**
Draft PR to test the random ID

### CODE REVIEW

File: scripts/svgr-configs/logos.ts [hunk 1]
```diff
@@ -70,8 +70,18 @@ export const LogosSvgReactOutPutConfig: Options = {
             },
           },
         },
+        {
+          name: 'prefixIds',
+          params: { prefix: Math.random().toString(16) },
```
Comment (shayan-deriv) [inline scripts/svgr-configs/logos.ts:75]: Let's use Nodejs Crypto for this.what do you think?

Comment (shayan-deriv) [reply scripts/svgr-configs/logos.ts:75]: there might be a chance that this will generate duplicated strings

Comment (shayan-deriv) [reply scripts/svgr-configs/logos.ts:75]:
```

import crypto from 'crypto'
const hash = crypto.randomBytes(16).toString('hex')
```

Comment (niloofar-deriv) [reply scripts/svgr-configs/logos.ts:75]: This approach doesn't help us fixing the ids issue:
https://github.com/svg/svgo/issues/674#issuecomment-285761453

**Pull request closed without merging**